### PR TITLE
Updated documentation for template binary_sensor

### PR DESCRIPTION
**Pull request by gadgetchnnel**
**Description:**
Added documentation for the new attribute_templates option for the template binary sensor.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#22664

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/9106"></a>

### CODE REVIEW

**Review by klaasnicolaas — DISMISSED** [review]

**Comment by klaasnicolaas** [comment]
Parent PR still open ⚠️ We have a merge conflict

**Review by frenck — CHANGES_REQUESTED** [review]
Hey, @gadgetchnnel, a wild merge conflict appeared 😢 

Could you please take a look and resolve it? Thanks already! 👍

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
Thanks for resolving the merge conflict so quickly @gadgetchnnel!
Awaiting parent PR now.

**Comment by frenck** [comment]
A wild merge conflict appeared again. @gadgetchnnel could you take a look? Thanks 👍

File: source/_components/binary_sensor.template.markdown [hunk 1]
```diff
@@ -210,6 +219,35 @@ binary_sensor:
 ```
 {% endraw %}
 
+### {% linkable_title Device Tracker sensor with Latitude and Longitude Attributes %}
```
Comment (klaasnicolaas) [inline source/_components/binary_sensor.template.markdown]:
```suggestion
### Device Tracker sensor with Latitude and Longitude Attributes
```

Comment (gadgetchnnel) [reply source/_components/binary_sensor.template.markdown]: Just made that change

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Parent PR got merged! 🎉 
Thanks for this @gadgetchnnel! 👍

**Pull request merged**